core/modules/wsched/ChunkState.cc

```diff
@@ -25,6 +25,7 @@
 
 // System headers
 #include <iterator>
+#include <iostream>
```

Comment:
better to swap order, we try to keep includes ordered alphabetically (within each group)